[DEV-904] refactor: SIEM Azion documentation for BB (#621)
aziontech/docs: src/content/docs/pt-br/pages/menu-principal/referencia/observe/siem/siem.mdx

permalink: /documentacao/produtos/siem-azion/
---

O **SIEM Azion** permite que você expanda a visibilidade dos acessos realizados as suas aplicações. Ao correlacionar dados disponíveis no [Real-Time Metrics](/pt-br/documentacao/produtos/real-time-metrics/) e no [Real-Time Events](/pt-br/documentacao/produtos/real-time-events/), você poderá ampliar as visões de segurança, volume de acessos, saúde e degradação.

## Dashboards

> Cada dashboard possui um conjunto de gráficos e tabelas que possibilitam acompanhar as suas aplicações conforme as diferentes necessidades de observabilidade. Selecionando diferentes períodos de tempo e utilizando os filtros disponíveis, você poderá ampliar a visão sobre as suas aplicações como refinar configurações de Edge Application, otimizar suas regras de Edge Firewall e WAF, identificar acessos de _bots_, entre outras. 

Reviewer: ```suggestion
Cada dashboard possui um conjunto de gráficos e tabelas que possibilitam acompanhar as suas aplicações conforme as diferentes necessidades de observabilidade. Selecionando diferentes períodos de tempo e utilizando os filtros disponíveis, você poderá ampliar a visão sobre as suas aplicações, como refinar configurações de Edge Application, otimizar suas regras de Edge Firewall e WAF, identificar acessos de *bots*, entre outras. 
```

## Alertas

> Além dos dashboards, o SIEM também disponibiliza a criação e acompanhamento de alertas. Os alertas podem ser visualizados na interface web e também podem ser enviados por e-mail, gerando notificações no início e no fim do evento.

Reviewer: ```suggestion
O SIEM também disponibiliza a criação e acompanhamento de alertas. Os alertas podem ser visualizados na interface web e também podem ser enviados por email, gerando notificações no início e no fim do evento.
```

> Os alertas podem ser configurados considerando um período de tempo, uma consulta utilizando um ou mais itens disponíveis no [Real-Time Events](/pt-br/documentacao/produtos/real-time-events/) e um limite esperado, como o valor mínimo, máximo, a média ou soma dos valores coletados.

Reviewer: ```suggestion
Os alertas podem ser configurados considerando um período de tempo, uma consulta utilizando um ou mais itens disponíveis no [Real-Time Events](/pt-br/documentacao/produtos/real-time-events/) e um limite esperado, como o valor mínimo, máximo, a média ou a soma dos valores coletados.
```

Exemplos de alertas que podem ser criados:

> - Considerando os últimos 5 minutos, gerar um alerta quando o volume de requisições por minuto for superior a 1000 requisições a qualquer uma das aplicaçÕes.

Reviewer: ```suggestion
- Considerando os últimos 5 minutos, gerar um alerta quando o volume de requisições por minuto for superior a 1.000 requisições a qualquer uma das aplicações.
```

## Dashboards

Cada dashboard possui um conjunto de gráficos e tabelas que possibilitam acompanhar as suas aplicações conforme as diferentes necessidades de observabilidade. Selecionando diferentes períodos de tempo e utilizando os filtros disponíveis, você poderá ampliar a visão sobre as suas aplicações como refinar configurações de Edge Application, otimizar suas regras de Edge Firewall e WAF, identificar acessos de _bots_, entre outras. 

## Alertas

> Além dos dashboards, o SIEM também disponibiliza a criação e acompanhamento de alertas. Os alertas podem ser visualizados na interface web e também podem ser enviados por e-mail, gerando notificações no início e no fim do evento.

Author: Conversando com o @arturdj , ele considerou redundante a parte `Além dos dashboards, o SIEM também disponibiliza a criação e acompanhamento de alertas.` . Portanto, deixo na revisão para verificar se remove-se ou não.
